.ci/Dockerfile

```diff
@@ -0,0 +1,107 @@
+ARG from=ubuntu:bionic
+FROM ${from}
+
+# Install tools and toolchain
+RUN apt-get update &&\
+    apt-get install -y --no-install-recommends \
+        wget \
+        software-properties-common \
+        apt-transport-https \
+        apt-utils \
+        gnupg2 \
+        nano \
+        &&\
+    wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | apt-key add - && \
+    apt-add-repository 'deb https://apt.kitware.com/ubuntu/ bionic main' &&\
+    apt-get update &&\
+    apt-get install -y --no-install-recommends \
+        git \
+        g++ \
+        g++-8 \
+        clang \
+        make \
+        cmake \
+        ninja-build \
+        ccache \
+        libgflags-dev \
+        python3-pip \
+        &&\
+    rm -rf /var/lib/apt/lists/*
+
+# Enable CCACHE by default
+ENV PATH=/usr/lib/ccache:$PATH
+RUN mkdir $HOME/.ccache &&\
+echo "max_size = 2.0G" > $HOME/.ccache/ccache.conf
+
+# Install swig
+RUN apt-get update &&\
+    apt-get install -y --no-install-recommends \
+        autotools-dev \
+        automake \
+        bison \
+        libpcre3-dev &&\
+    rm -rf /var/lib/apt/lists/* &&\
+    cd /tmp/ &&\
+    git clone --depth 1 -b rel-4.0.0 https://github.com/swig/swig.git &&\
+    cd swig &&\
+    sh autogen.sh &&\
+    ./configure &&\
+    make -j8 &&\
+    make install &&\
+    rm -r /tmp/swig
+
+# Install ignition robotics
+RUN echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" \
+        > /etc/apt/sources.list.d/gazebo-stable.list &&\
+    echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-prerelease `lsb_release -cs` main" \
+        > /etc/apt/sources.list.d/gazebo-prerelease.list &&\
+    echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-nightly `lsb_release -cs` main" \
```

Comment:
I thought just  adding `ubuntu-nightly` should  be enough.

Reply:
I thought it was enough too, but the repo is not self contained and nightly packages depend on other that are contained only in the other two. Not sure if this was intentional or just a packaging mistake. For the time being I'm fine to keep them all enabled, I wish to switch as soon as possible to using only the stable channel.